### PR TITLE
Release 4.7.0

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-release-4.7.0](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-release-4.7.0)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-release-4.7.0](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-release-4.7.0)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.7.0 (2020-10-26)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:1]:
```suggestion
# 4.7.0 (2020-10-27)
```

**Review by blackbaud-johnly — APPROVED** [review]
Added one comment, but not sure if we really need to update the date.

**Pull request merged**